### PR TITLE
https://github.com/escapestudios-cookbooks/newrelic/issues/227

### PR DESCRIPTION
**Pull request by gswallow**
Pull request submitted.  I think I may just use the LWRP though :)

Thanks!

### CODE REVIEW

File: providers/server_monitor.rb [hunk 1]
```diff
@@ -35,7 +35,12 @@ def install_newrelic_service_linux
   package new_resource.service_name do
     action new_resource.action
   end
+
   # configure your New Relic license key
+  if node.platform_family.include?('debian')
+    execute "dpkg-divert --add --local --rename #{new_resource.config_path}/nrsysmond.cfg"
+  end
+
```
Comment (djoos) [inline providers/server_monitor.rb]: Would you mind explaining a bit more in detail why this is needed for the browser_monitoring_capture_attributes-change?

If possible, revert this bit and then I'm happy to merge this PR in!

**Comment by gswallow** [comment]
I have no idea.  This pull request is from 2014.

**Comment by djoos** [comment]
Your actual commit is from 2014, however you "only" opened the PR in January ;-)

No worries - I'll have a deeper look at some point...
Thanks!

**Comment by gswallow** [comment]
If I recall, I had issues where my nrsysmond.cfg file was overwritten when the package gets updated.

That was my attempt to block the overwrite.  I don't think it worked.

We aren't really using Chef for that anymore, anyway, but I removed it.

**Pull request closed without merging**